### ISSUE TITLE
Server & Client crashing on startup.

### ISSUE THREAD

**Issue by Ozykz** [MJRLegends/Space-Astronomy-Feedback-#288]
Trying to get a solution for a server and client crashing.

I went through the list, but couldn't find a relevant solution for my issue, 

I tried deleting the HQM folder,
I tried running the mod from the FTB launcher, 
I haven't disabled the opencomputers mod because when I do It doesn't allow me to load the pack,  
I've disabled the chickenloaders mod, with no effect.
Also reverted Immersive Engineering to 0.7.4

This is my server side crash log: http://pastebin.com/sYq4P73g
Client side crashlog: http://pastebin.com/sMJFu4L5

**Comment by Ozykz** [comment]
Solved it,

I had to delete both Chickenloaders and EnhancedPortals, then load the game so that it deleted all items relating to both those mods, then I could re-enable those mods and now It's working, just as before.

It definitely had to be both, I tried one and than the other on it's own, and I still didn't work.

I imagine I should avoid using both of those mods, so for now, I will. I don't know if this is a fluke occurrence or an actual bug that needs to be investigated/fixed.

Cheerio!
